pom.xml

```diff
@@ -37,10 +37,10 @@
 		<module>fhir-resource-wih</module>
 		<module>cdshooks-wih</module>
 		<module>web-util</module>
+		<module>fhir-query-helper-base</module>
```

Comment:
this is mainly a different issue: we were building the parent project first for fhir-query-helpers before building the children